core-web/libs/block-editor/src/lib/extensions/action-button/actions-menu.extension.ts

```diff
@@ -1,16 +1,19 @@
+import { flatten } from 'lodash';
 import { PluginKey } from 'prosemirror-state';
 import { Subject } from 'rxjs';
 import tippy, { GetReferenceClientRect } from 'tippy.js';
 
 import { ComponentRef, ViewContainerRef } from '@angular/core';
 
-import { filter, take, takeUntil } from 'rxjs/operators';
+import { filter, take } from 'rxjs/operators';
 
 import { Editor, Extension, Range } from '@tiptap/core';
 import { FloatingMenuPluginProps } from '@tiptap/extension-floating-menu';
 import { Level } from '@tiptap/extension-heading';
 import Suggestion, { SuggestionOptions, SuggestionProps } from '@tiptap/suggestion';
 
+import { CustomBlock } from '@dotcms/dotcms-models';
```

Comment:
I think the name should be more like: "RemoteCustomExtentions" or something very clear.